R/Editor.R

```diff
@@ -0,0 +1,108 @@
+editor_ui <- function(id, x) {
```

Comment:
Should we rename functions to `teal` convention of `ui_editor` instead of `editor_ui`

This is not only for this function / file but across reporter

tagging @m7pr and @gogonzo, but other feel free to chime in